ipynb/energy/EnergyModel_ClusterEnergy.ipynb

```diff
@@ -106,12 +106,12 @@
     "    \"emeter\" : {\n",
     "        'instrument' : 'hwmon',\n",
     "        'conf' : {\n",
-    "            'sites' : [ 'a53', 'a57' ],\n",
+    "            'sites' : [ 'BOARDLITTLE', 'BOARDBIG' ],\n",
```

Comment:
Thanks @bjackman for testing on R0! :+1: 
@cloverpop the real issue is that in most recent kernel there is not more need of the  juno/juno2 distinction, can you refresh this PR to be a patch which removes that distinction?